buildengine/engine.go

```diff
@@ -30,8 +30,6 @@ type schemaChange struct {
 }
 
 type Listener struct {
-	OnBuildComplete  func(project Project, err error)
-	OnDeployComplete func(project Project, err error)
 }
```

Comment:
We can probably remove the whole Listener thing here since we're not using it anymore.

Reply:
cool, wasn't sure if we were still going to use with a new function to refresh diagnostics with in your change but will remove!